### PR TITLE
DOCS-6980 Removed download link for API Analytics Handlers

### PR DESCRIPTION
**Pull request by hannanelson**
Cleaned up the intro up to the download section, added icons, addressed bug, and updated See Also links.

The bug fix is on lines 36-44 of the analytics topic.

### CODE REVIEW

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 1]
```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
...
+Runtime Manager agent sends messages from Mule runtime engine.
+To send event information to third-party software, you must have version 1.3.0 or later of Runtime Manager agent.
 
-For help installing the agent, see xref:installing-and-configuring-runtime-manager-agent.adoc[Installing and Configuring Runtime Manager Agent].
+For information about updating the agent, see xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update Runtime Manager Agent].
 
 == Install API Analytics Tracking Database Internal Handlers
 
-Download http://mule-agent.s3.amazonaws.com/1.5.1/mule-agent-internal-handlers-db-1.5.1.zip[the API Analytics Tracking Database Internal Handlers] and unzip them in the `{MULE_HOME}/plugins/mule-agent-plugin/lib/modules` folder.
+Depending on your version of Mule agent, you might need to install the API Analytics Tracking Database Internal Handlers ZIP file.
```
Comment (bahui80) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: I am not sure about this, @mcocaro do you know about this file???

Comment (hannanelson) [reply modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: @bahui80 - that's what it was called before (see the text in [] in the red).

Comment (bahui80) [reply modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: OK great

**Review by bahui80 — APPROVED** [review]
LGTM

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc (hunk 1 above)
Comment (mcocaro) [reply modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: yes - Mule Agent 1.14.3 and 2.4.5 will include these bundled by default so no need for extra steps. Prior versions require a manual download step.

**Review by mcocaro — APPROVED** [review]
LGTM

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 2]
```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-The HTTP API analytics produced by applications running on the API gateway runtime can be easily stored in a database by configuring a few settings.
+You can configure Runtime Manager agent to store HTTP API analytics data from Mule apps running on the API gateway runtime in a database.
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: That's one tall drink of water of a sentence!
Can you reword a bit to clarify what "in a database" modifies: "store" or "runtime"? "...to store in a database HTTP API analytics data..."?

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 3]
```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
...
 
-[NOTE]
-====
-Sending data to third party software is a feature that is currently only available for applications deployed to local servers. It is not available for applications that you deploy via the Runtime Manager to xref:cloudhub.adoc[CloudHub]. See xref:deployment-strategies.adoc[Deployment Strategies].
-====
+Sending data to third-party software is only available for applications deployed to local servers.
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]:
```suggestion
Sending data to third-party software is available only for applications deployed to local servers.
```

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 4]
```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
...
+
+* API gateway standalone version 2.1.0 or later
+* Runtime Manager agent version 1.3.0 or later
 
-== Install the Runtime Manager Agent
+== Install or Update Runtime Manager Agent
 
-The Runtime Manager agent is in charge of sending messages out of Mule. To send event information to third party software, you must have version 1.3.0 or later of the Runtime Manager agent.
+Runtime Manager agent sends messages from Mule runtime engine.
+To send event information to third-party software, you must have version 1.3.0 or later of Runtime Manager agent.
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: Do we need to repeat the prereq here?

Comment (kbostrom) [reply modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: Also...why have a topic that is little more than a cross-ref?

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 5]
```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
...
 
-For help installing the agent, see xref:installing-and-configuring-runtime-manager-agent.adoc[Installing and Configuring Runtime Manager Agent].
+For information about updating the agent, see xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update Runtime Manager Agent].
 
 == Install API Analytics Tracking Database Internal Handlers
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: This needs a hyphen or two, perhaps Install API Analytics-Tracking Database Internal Handlers?
Or rewording.

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc (hunk 1 above)
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: Mule agent?
Search for GLOBALLY in this file and reword. It's either Mule or (Anypoint) Runtime Manager agent.

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 6]
```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
...
 
-Download http://mule-agent.s3.amazonaws.com/1.5.1/mule-agent-internal-handlers-db-1.5.1.zip[the API Analytics Tracking Database Internal Handlers] and unzip them in the `{MULE_HOME}/plugins/mule-agent-plugin/lib/modules` folder.
+Depending on your version of Mule agent, you might need to install the API Analytics Tracking Database Internal Handlers ZIP file.
+
+For Mule agent versions later than 1.14.3 and 2.4.5, you don't need to install the ZIP file.
+The required JAR files are included in the Mule agent plugin.
+
+For Mule agent versions earlier than 1.14.3 and 2.4.5:
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]: This covers later and earlier, but what about those exact versions? Reword to clarify.

File: modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc [hunk 7]
```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
...
+
+. Request the API Analytics Tracking Database Internal Handlers ZIP file (`mule-agent-internal-handlers-db-1.5.1.zip`) from Support.
+. Extract the ZIP file to the `{MULE_HOME}/plugins/mule-agent-plugin/lib/modules` folder.
+
 
 == Agent Configurable Fields
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc]:
```suggestion
== Agent-Configurable Fields
```

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 8]
```diff
@@ -5,38 +5,45 @@ endif::[]
...
-By simply configuring a few settings, the event notifications produced by your running Mule applications can be easily stored in a database.
 
-[NOTE]
-====
-Sending data to third-party software is a feature that is currently only available for applications deployed to local servers. It is not available for applications that you deploy via the Runtime Manager to xref:cloudhub.adoc[CloudHub]. See xref:deployment-strategies.adoc[Deployment Strategies]xref:deployment-strategies.adoc[Deployment Strategies].
+You can configure Runtime Manager agent to store the event notifications from Mule apps in a database.
 
-It's not available either for applications deployed to Pivotal Cloud Foundry (PCF).
-====
+Sending data to third-party software is only available for applications deployed to local servers.
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc]:
```suggestion
Sending data to third-party software is available only for applications deployed to local servers.
```

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 9]
```diff
@@ -5,38 +5,45 @@ endif::[]
...
-* The Runtime Manager agent plugin version 1.3.0 or above
+Ensure that the following software is installed:
 
-== Install the Runtime Manager Agent
+* Mule version 3.6 or later
+* API gateway standalone version 2.1.0 or later
+* Runtime Manager agent version 1.3.0 or later
 
-First you must install the Runtime Manager Agent, which is in charge of sending messages out of Mule. In order to send event information to third party software, you need to have version 1.3.0 or later of the Runtime Manager Agent.
+== Install or Update Runtime Manager Agent
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc]: DITTO previous edits to this same text earlier.

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 10]
```diff
@@ -5,38 +5,45 @@ endif::[]
...
 
-See xref:installing-and-configuring-runtime-manager-agent.adoc[installing and configuring Runtime Manager Agent].
+Runtime Manager agent sends messages from Mule runtime engine.
+To send event information to third-party software, you must have version 1.3.0 or later of Runtime Manager agent.
 
+See xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update Runtime Manager Agent].
 
 == Install Event Tracking Database Internal Handlers
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc]: DITTO previous edits to this same text earlier.

File: modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc [hunk 11]
```diff
@@ -5,38 +5,45 @@ endif::[]
...
+For Mule agent versions later than 1.14.3 and 2.4.5, you don't need to install the ZIP file.
+The required JAR files are included in the Mule agent plugin.
+
+For Mule agent versions earlier than 1.14.3 and 2.4.5:
+
+. Request the Event Tracking Database Internal Handlers ZIP file (`mule-agent-internal-handlers-db-1.5.1.zip`) from Support.
+. Extract the ZIP file to the `{MULE_HOME}/plugins/mule-agent-plugin/lib/modules` folder.
+
 
 == Agent Configurable Fields
```
Comment (kbostrom) [inline modules/ROOT/pages/sending-event-data-from-arm-to-db.adoc:51]: ditto

**Review by kbostrom — APPROVED** [review]
Done!

**Pull request merged**